harmonizer/release_types.test.ts

```diff
@@ -84,6 +84,55 @@ describe('release types', () => {
 				title,
 				new Set(['Soundtrack']),
 			])),
+			// Remix releases
+			...([
+				'Human (Paul Woolford Remix)',
+				'Paper Romance (Purple Disco Machine Remix - Edit)',
+				'Paper Romance (Purple Disco Machine Remix) [Edit]',
+				'Paper Romance (Purple Disco Machine Remix) (Edit)',
+				'Paper Romance (Purple Disco Machine Remix; Edit)',
+				"Stay (Don't Go Away) [feat. Raye] [Nicky Romero Remix]",
+				'Anti‐Hero (Kungs remix extended version)',
+				'Remix',
+				'Anti‐Hero (Remixes)',
+				'The One (feat. Daddy Yankee) [The Remixes]',
+				'The Remixes',
+				'The Remixes - Vol.1',
+				'The Remixes, Pt. 1',
+				'Remixes',
+				'Remixes 81>04',
+				'Never Say Never - The Remixes',
+				'Skin: The Remixes',
+				'The Hills Remixes',
+				'MIDI Kittyy - The Remixes Vol 1',
+				'The Slow Rush B-Sides & Remixes',
+				'Remixed',
+				'Remixed (2003 Remaster)',
+				'Remixed Sides',
+				'Remixed: The Definitive Collection',
+				'The Hits: Remixed',
+				'Remixed & Revisited',
+				'Revived Remixed Revisited',
+				'Welcome To My World (Remixed)',
+				'Mörkrets Narr Remixed',
+			].map((
+				title,
+			): FunctionSpec<typeof guessTypesFromTitle>[number] => [
+				`should detect remix type (${title})`,
+				title,
+				new Set(['Remix']),
+			])),
```

Comment:
There are so many examples of remix (and soundtrack) releases now, that I'm wondering whether we should factor these out of the main array in order to make the test output less verbose.
It probably makes sense to independently register only one (or a few) tests with `it` that contain a loop of assertions.
I would do that outside of this PR, what do yout think?